### PR TITLE
Add revision table

### PR DESCRIPTION
**Pull request by BetsyMcPhail**
### Fields that belong to publication
* Title
* Abstract
* Type
* Comments
* Statistics (Views & aggregate downloads)
* Handle link (not editable)
* Approval status
* Permission (i.e. is a private submission)

### Fields that belong to revision
* Submission date (not editable)
* Submitter (not editable)
* Author(s)
* Institution
* Revision notes
* Revision version (new)
* Keywords (Tags)
* Grant/Tracking Number
* Disclaimer
* Categories
* Resources (files/github and thumbnail logo)
* License
* Review / Certification level
* Export citation link (contains date)

### Unknown fields
* Related work --> is this displayed anywhere?
* Copyright holder -->  is this displayed anywhere?

### CODE REVIEW

**Comment by BetsyMcPhail** [comment]
@josephsnyder this is ready for a first review.

**Comment by BetsyMcPhail** [comment]
@josephsnyder I amended the original commits to include the changes we discussed. Let me know what you think!

File: public/css/view/view.index.css [hunk 1]
```diff
@@ -190,3 +190,11 @@ div.Panel h4{
 opacity: 0.5!important;
 filter: Alpha(Opacity=50)!important;
 }
+
+td {border: 1px #DDD solid; padding: 5px; }
+tr.clickable-row { cursor: pointer; s}
```
Comment (josephsnyder) [inline public/css/view/view.index.css]: Extra "s" here.  But it doesn't have an effect on the page,

**Review by josephsnyder — COMMENTED** [review]
One character to remove but it works perfectly otherwise.  Great job!

**Pull request merged**